tests/queries/0_stateless/03156_group_concat.sql

```diff
@@ -42,6 +42,19 @@ SELECT groupConcat(',', 3, 3)(number) FROM numbers(10); -- { serverError TOO_MAN
 
 SELECT length(groupConcat(number)) FROM numbers(100000);
 
+SELECT 'TESTING GroupConcat second argument overload';
+
+TRUNCATE TABLE test_groupConcat;
+
+INSERT INTO test_groupConcat VALUES (0, 95, 'abc', [1, 2, 3]), (1, NULL, 'a', [993, 986, 979, 972]), (2, 123, 'makson95', []);
+
+SELECT groupConcat(',', p_int) FROM test_groupConcat;
+SELECT groupConcat('.')(p_string) FROM test_groupConcat;
+SELECT groupConcat('/', p_array) FROM test_groupConcat;
+
+SELECT group_concat('/', p_array) FROM test_groupConcat;
+SELECT grouP_CONcat('/', p_array) FROM test_groupConcat;
```

Comment:
Could you add a test verifying that 'arguments delimiter' override 'delimiter parameters'?

Reply:
Yes, I will, thanks for noticing